payment/utils.py

```diff
@@ -68,6 +69,47 @@ def send_confirmation_email(user, order_number):
     email.send()
 
 
+def register_user_verified_cohort(user, order):
+    """
+    Once a user has been verified, add it to a cohort.
+
+    The cohort to which the user should be added is given by the VERIFIED_COHORTS and the DEFAULT_VERIFIED_COHORT
+    settings. VERIFIED_COHORTS is a dict of the form:
+
+        {
+            "<course id>": "<cohort name>",
+        }
+
+    If the course ID is not present in VERIFIED_COHORTS, the cohort name is given by DEFAULT_VERIFIED_COHORT.
+    If a course is associated to a cohort that doesn't exist or an empty cohort name, then the user is not added to any
+    cohort. Thus, to disable this feature for a specific course, set:
+
+        VERIFIED_COHORTS = {
+            "<course id>": None,
+        }
+
+    To disable this feature globally, set:
+
+        VERIFIED_COHORTS = {}
+        DEFAULT_VERIFIED_COHORT = None
+    """
```

Comment:
@sampaccoud I'm following up on our [previous conversation](https://github.com/openfun/fun-apps/pull/694/files#r492082568). I think this implementation strikes a good compromise: there is a single `DEFAULT_VERIFIED_COHORT` that defines the name of the cohort to which users should be added. This setting can be overridden individually for every course by adding an entry to the `VERIFIED_COHORTS` setting. And if the cohort does not exist, users are simply not added to them.

The `CourseUserGroup` model unfortunately has no other field than the `course_id` and `name` which we can use.

Reply:
ok. I follow you.

Reply:
Yes, that's right, the issue with CourseUserGroup makes it impossible to compare to anything else than a "name". Great approach !

Reply:
@sampaccoud : any chance this can be reviewed and merged into the main fun-apps code now ?